docs/admin/admin-ui-setup/credentials-setup.md

```diff
@@ -2,34 +2,56 @@
 
 Credentials are used to unlock protected resources such as applications, containers, and other assets.
 
-## Types of credentials
-
-The Credential manager in the Run:ai environment supports 4 types of credentials:
+The *Credentials* manager in the Run:ai environment supports 4 types of credentials:
 
 1. Docker registry.
 2. Access key.
 3. User name and password.
-4. [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret){target=_blank}.
+<!-- 4. [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret){target=_blank}.
+-->
 
-## Configuring Credentials
+## Secrets
+
+Credentials are built using `Secrets`. A `Secret` is an object that contains a small amount of sensitive data so that you don't need to include confidential data in your application code. When creating a credential you can either **create a new secret** or use an [existing secret](#existing-secrets).
```

Comment:
```suggestion
Credentials are built using `Secrets`. A `Secret` is an object that contains a small amount of sensitive data so that you don't need to include confidential data in your application code. When creating a credential you can either **create a new secret** securely with Run:ai or use an [existing secret](#existing-secrets).
```